content/blog/2025/08/2025-08-22-twim.md

```diff
@@ -0,0 +1,199 @@
+
+> The German Federal Office for Information Security (BSI) is currently running an online survey on the Cyber Resilience Act (CRA) via its "Dialogue for Cybersecurity".
+>
+> The aim is to better understand the impact of the CRA on open source and to develop practical guidelines based on community input.
+>
+> The survey is aimed at three groups:
+>
+> * Manufacturers
+> * Open source projects
+> * Potential open source stewards
+>
+> The survey is anonymous and open until 31 August 2025.
+>
+> Responses received by 30 July 2025 were included in an initial evaluation [presented](https://programm.froscon.org/froscon2025/talk/0a656836-cb27-4ef1-80e1-d64c553a96ca/) at FrOSCon 2025.
+>
+> ### Questionnaires
+>
+> German
+>
+> * [Manufacturers (DE)](https://dialog-cybersicherheit.limesurvey.net/552671?lang=de)
+> * [Open source projects (DE)](https://dialog-cybersicherheit.limesurvey.net/323741?lang=de)
+> * [Potential open source stewards (DE)](https://dialog-cybersicherheit.limesurvey.net/415696?lang=de)
+>
+> English
+>
+> * [Manufacturers (EN)](https://dialog-cybersicherheit.limesurvey.net/582853?lang=en)
+> * [Open source projects (EN)](https://dialog-cybersicherheit.limesurvey.net/241948?lang=en)
+> * [Potential open source stewards (EN)](https://dialog-cybersicherheit.limesurvey.net/146965?lang=en)
+>
+> ### Why this matters for Matrix
+>
+> The CRA will influence how open source software is developed, distributed, and maintained across Europe.
+>
+> For the Matrix ecosystem, this includes everything from server implementations (e.g. Synapse, Dendrite), to client apps, to bridges.
+>
+> By participating in the survey, the Matrix community can ensure its perspective is heard and that resulting guidelines enable sustainable, community-led, open collaboration.
+>
+> If you maintain, contribute to, or rely on Matrix projects, please consider filling out the survey and share it with others in our community.
+
+## Matrix Federation Stats
+
+[Aine [etke.cc]](https://matrix.to/#/@aine:etke.cc) reports
+
+> collected by [MatrixRooms.info](https://matrixrooms.info/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats) - an [MRS](https://github.com/etkecc/mrs) instance by [etke.cc](https://etke.cc?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats)
+>
+> As of today, `12975` Matrix federateable servers have been discovered by matrixrooms.info, `3724` (`28.7%`) of them are publishing their rooms directory over federation.
+> The published directories contain `17028` rooms.
+>
+> Stats timeline is available on [MatrixRooms.info/stats](https://matrixrooms.info/stats/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats)
+>
+> [How to add your server](https://matrixrooms.info/indexing/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats) | [How to remove your server](https://matrixrooms.info/deindexing/?utm_source=twim&utm_medium=matrix&utm_campaign=federation-stats)
+
+## Dept of Ping
```

Comment:
Ping section seems like its missing the table?

Reply:
added in d3c7e09